operator_ui/@types/use-react-hooks.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'use-react-hooks'
```

Comment:
I'm surprised this was needed. Couldn't `import` the type where necessary? Also, why are we still on the old react?

Reply:
There are no type definitions for this library:

- https://www.npmjs.com/search?q=%40types%2Fuse-react-hooks
- https://github.com/tannerlinsley/use-react-hooks

Tried upgrading a few months ago but enzyme didn't support hooks yet so punted at the time https://github.com/airbnb/enzyme/issues/2011. I'd like to do another push to get workspaces working + our dependencies updated now that we don't have the deadline for mainnet. Getting workspaces working is schedule for 2 weeks time.